colibre/scripts/bh_accretion_evolution.py

```diff
@@ -16,6 +16,7 @@
 from swiftpipeline.argumentparser import ScriptArgumentParser
 
 from velociraptor.observations import load_observation
+from velociraptor.observations import load_observations
```

Comment:
This line is a duplicate of the line above
```suggestion
```

Reply:
Thanks, hadn't noticed it!